### PR TITLE
Also transform webhooks' clientConfig service reference

### PR DESCRIPTION
**Pull request by evankanderson**
Added tests.

### CODE REVIEW

File: yaml.go [hunk 1]
```diff
@@ -136,6 +136,9 @@ func decode(reader io.Reader) ([]unstructured.Unstructured, error) {
 
 // isURL checks whether or not the given path parses as a URL.
 func isURL(pathname string) bool {
+	if _, err := os.Lstat(pathname); err == nil {
```
Comment (jcrossley3) [inline yaml.go:139]: Is this necessary on Windows or something?

Comment (evankanderson) [reply yaml.go:139]: Without this, the "absolute path" test in `yaml_test.go` will try to interpret "c:\Users\...." as a URL with `Scheme = "c"` and some other garbage.

I can drop this if you want; I'm not sure what the best way to distinguish between file paths on Windows + Unix and URIs would be.

Comment (jcrossley3) [reply yaml.go:139]: It's fine. I was just curious.

**Comment by evankanderson** [comment]
Note, this fixes https://testgrid.knative.dev/serving-operator#integration-tests-on-latest-serving

**Comment by evankanderson** [comment]
https://github.com/knative/pkg/pull/843 was the location where the initial debugging happened, but it looks like there was no issue opened.

**Pull request merged**